desktop/ui/src/main/java/org/datacleaner/widgets/visualization/JobGraphMouseListener.java

```diff
@@ -218,7 +221,16 @@ public void actionPerformed(ActionEvent e) {
             popup.add(createLinkMenuItem(componentBuilder));
         }
 
-        popup.add(new RenameComponentMenuItem(componentBuilder));
```

Comment:
Please check if RenameComponentMenuItem is ever used now... If not, then delete the class.